LanguageFeatures/Patterns/type_inference_A08_t01.dart

```diff
@@ -43,10 +45,10 @@ main() {
     v2.expectStaticType<Exactly<A>>();
     v3.expectStaticType<Exactly<A>>();
   }
+  String log = "";
   try {
-    var <String, A>{"key1": A v1, "key2": B v2, "key3": v3} = {};
-    v1.expectStaticType<Exactly<A>>();
-    v2.expectStaticType<Exactly<A>>();
-    v3.expectStaticType<Exactly<A>>();
+    var <String, A>{"key1": A v1, "key2": B v2, "key3": v3} =
```

Comment:
I think `B v2` is a compile-time error because this is in a declaration context.